### PR TITLE
Surface AgentExecutor kwargs for toolkits

### PR DESCRIPTION
**Pull request by zachschillaci27**
Surface separate `kwargs` for `Agent` and `AgentExecutor` in a consistent way for the agent toolkits. Following the example in `create_pbi_agent` and `create_pbi_chat_agent`,`agent_kwargs` and `kwargs` are now accessible for all `AgentExecutor` creator functions. This allows the user to separately control the `Agent` and `AgentExecutor` parameters. 

**NOTE:** This will modify the behavior for some of the existing toolkits.

### CODE REVIEW

**Review by hwchase17 — COMMENTED** [review]
since (as you noted) this changes behavoir for a good amount of toolkits, id actually prefer to keep `kwargs` to agent, and add `agent_executor_kwargs` as a dictionary (and update the pbi toolkit, since that would just be one toolkit changed, rather than the 6 here

**Comment by zachschillaci27** [comment]
> since (as you noted) this changes behavoir for a good amount of toolkits, id actually prefer to keep `kwargs` to agent, and add `agent_executor_kwargs` as a dictionary (and update the pbi toolkit, since that would just be one toolkit changed, rather than the 6 here

Good point, I had considered doing the same originally. Let me change it

File: langchain/agents/agent_toolkits/powerbi/chat_base.py [hunk 1]
```diff
@@ -43,11 +45,12 @@ def create_pbi_chat_agent(
         llm=llm,
         tools=tools,
         system_message=prefix.format(top_k=top_k),
-        user_message=suffix,
```
Comment (zachschillaci27) [inline langchain/agents/agent_toolkits/powerbi/chat_base.py:46]: It looks like this may have been a bug?

Comment (zachschillaci27) [reply langchain/agents/agent_toolkits/powerbi/chat_base.py:46]: @hwchase17 please confirm if you have a chance

Comment (hwchase17) [reply langchain/agents/agent_toolkits/powerbi/chat_base.py:46]: yup think youre right!

**Review by hwchase17 — COMMENTED** [review]
lgtm, i can help squash merge conflicts

**Comment by zachschillaci27** [comment]
> lgtm, i can help squash merge conflicts

thanks!

**Pull request merged**